weave/trace_server/sqlite_trace_server.py

```diff
@@ -1097,11 +1097,16 @@ def _make_derived_summary_map(
     status = make_call_status_from_exception_and_ended_at(exception, ended_at_dt)
     latency = None if not ended_at_dt else (ended_at_dt - started_at_dt).microseconds
     display_name = display_name or op_name_simple_from_ref_str(op_name)
+    summary = summary_dump or {}
```

Comment:
this is an ugly artifact of these typing shanagans - i think we should punt on that instead of adding all these nulls over the wire